admin/cluster_containers_setup.py

```diff
@@ -0,0 +1,369 @@
+# Copyright ClusterHQ Inc.  See LICENSE file for details.
+import yaml
+import os
+import sys
+from copy import deepcopy
+from json import dumps
+from itertools import repeat
+from ipaddr import IPAddress
+
+from treq import json_content
+from eliot import add_destination, Message
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.python.filepath import FilePath
+from twisted.python import usage
+from twisted.web.http import OK
+
+from flocker.common import loop_until
+from flocker.control.httpapi import REST_API_PORT
+from flocker.apiclient import FlockerClient
+from flocker.ca import treq_with_authentication
+
+
+def eliot_output(message):
+    """
+    Write pretty versions of eliot log messages to stdout.
+    """
+    message_action = message.get('action')
+
+    if message_action is not None:
+        msg = "%s\n" % message_action
+        sys.stdout.write(msg)
+        sys.stdout.flush()
+
+
+class ContainerOptions(usage.Options):
+    """
+    Parses the options passed as an argument to the create container script.
+    """
+    description = "Set up containers in a Flocker cluster."
+
+    optParameters = [
+        ['apps-per-node', None, 0, 'Number of application containers per node',
```

Comment:
A default of 0 made sense when this code was part of `setup-cluster`. Now that starting containers is the only purpose of this script, make the default 1.